trajectory_executor/src/trajectory_executor/trajectory_executor.cpp

```diff
@@ -94,7 +80,7 @@ namespace trajectory_executor
 
         if (_cur_traj != nullptr) {
             if (_timesteps_since_last_traj > 0) {
-                _cur_traj = std::unique_ptr<cav_msgs::TrajectoryPlan>(new cav_msgs::TrajectoryPlan(trimPastPoints(*_cur_traj)));
+                _cur_traj = std::unique_ptr<cav_msgs::TrajectoryPlan>(new cav_msgs::TrajectoryPlan(*_cur_traj));
```

Comment:
Can't we just delete this whole `if`-block? This currently basically just does `_cur_traj = *&_cur_traj` which I don't think is doing much for us here.